test/end-to-end.js

```diff
@@ -970,6 +976,44 @@ describe('end-to-end', function() {
       });
     });
 
+    it('shows error behavior responses in development mode', function(done) {
+      var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+      var bootOptions = {
+        appRootDir: SANDBOX,
+        env: 'development'
+      };
+      boot(app, bootOptions, function(err) {
+        if (err) return done(err);
+        app.listen(0, function() {
+          request(app)
+            .get('/url-does-not-exist')
+            .end(function(err, res) {
+              expect(res.error.text.match(/url-does-not-exist/));
+              done(err);
+          });
+        });
+      });
+    });
+
+    it('shows error behavior responses in production mode', function(done) {
```

Comment:
The test name should describe what is the expected behaviour/outcome. E.g. "it omits sensitive error details in production mode".